set.mm

```diff
@@ -65959,14 +65966,20 @@ complete lattice has a (least) fixpoint.  Here we specialize this
 
   $( Cantor's theorem fails for the universal class (which is not a set but a
      proper class by ~ vprc ).  Specifically, the identity function maps the
-     universe onto its power class.  Compare ~ canth that works for sets.  See
-     also the remark in ~ ru about NF, in which Cantor's theorem fails for sets
-     that are "too large."  This theorem gives some intuition behind that
-     failure: in NF the universal class is a set, and it equals its own power
-     set.  (Contributed by NM, 29-Jun-2004.) $)
+     universe onto its power class.  Compare ~ canth that works for sets.
+
+     This failure comes from a limitation of the collection principle (which is
+     necessary to avoid Russell's paradox ~ ru ): ` P _V ` , being a class,
+     cannot contain proper classes, so ` ~P _V ` , so it is no larger than
+     ` _V ` , which is why ` _I ` "succeeds" in being surjective onto ` ~P _V `
+     (see ~ pwv ).  See also the remark in ~ ru about NF, in which Cantor's
```

Comment:
I think this (and the similar remark in ~ pwv) are necessary to avoid possible misunderstandings from a new reader, and to hint at the root cause of the "problem", namely, the necessary restriction of the collection principle to avoid Russell's paradox in Zermelo-style set theories.